### DISCUSSION TITLE
Mouse sensitivity settings for the android version (if posible this is)

### DISCUSSION THREAD

**Discussion by hamax2468** [ppy/osu#14043] [Ideas] [answered]
So a little while ago I installed the lazer client on my new s10 and coming from regular osu (i installed lazer on my phone for portability) it was pretty good, however after getting a c to USB dongle I noticed that i couldn’t change my sensitivity, the default on lazer is trash and I already tried system sensitivity but on click the mouse switches to the game’s default sensitivity, so I looked into it and found out it was disabled for some reason on all platforms except windows, however all of these posts mentioned that it was just greyed out and not entirely missing. So going back to the topic, is there a way this could be re-added or given a debug option or something?
Kind regards
A random osu player

PS: before this post is removed for being in the wrong category I wasn’t sure what to flag it as, weather a bug or suggestion.

**Comment by smoogipoo** [comment, marked as answer]
This may be possible after we get android/iOS to use SDL2, but unsure.

**Reply by peppy** [reply]
This has been asked before. Android does not provide an API for this (nor does iOS). And as far as I recall, there was also no way to hide the cursor.

Will probably have to wait until if/when mobile operating systems see mice as gaming input devices, not just pointing devices.

**Reply by hamax2468** [reply]
> This has been asked before. Android does not provide an API for this (nor does iOS). And as far as I recall, there was also no way to hide the cursor.
> 
> Will probably have to wait until if/when mobile operating systems see mice as gaming input devices, not just pointing devices.

Thanks for informing me, I wasn’t exactly sure what was going on with it but oddly enough you can configure the sensitivity in android’s settings but in game whenever I clicked it would switch over to the ig sensitivity.
Edit: also my apologies peppy for the accidental spam across the 2 threads this discussion was taking place on (this one and the ancient one from 2020)
